### PR TITLE
Feature/noc/eui 2458 navigation aa

### PR DESCRIPTION
**Pull request by adnanakgun**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/noc/containers/noc-home/noc-home.component.ts [hunk 1]
```diff
@@ -13,4 +13,16 @@ export class NocHomeComponent {
     private store: Store<fromFeature.State>,
   ) { }
 
+  public back(event) {
+    console.log('back event triggered ', event);
```
[the remark this answers is not in the document]
Comment (adnanakgun) [reply src/noc/containers/noc-home/noc-home.component.ts:17]: It's a place holder for now

File: src/noc/store/actions/noc-navigation.action.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Action } from '@ngrx/store';
+
+export const CHANGE_NAVIGATION = '[NOC Navigation] Change Navigation';
+
+export class ChangeNavigation implements Action {
+    readonly type = CHANGE_NAVIGATION;
+    constructor(public payload: any) {}
```
Comment (udaydenduluri33) [inline src/noc/store/actions/noc-navigation.action.ts:7]: Could this be typed instead of any?

**Review by udaydenduluri33 — APPROVED** [review]
LGTMa

File: src/noc/containers/noc-home/noc-home.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply src/noc/containers/noc-home/noc-home.component.ts:17]: Ok

File: src/noc/store/actions/noc-navigation.action.ts (hunk 2 above)
Comment (adnanakgun) [reply src/noc/store/actions/noc-navigation.action.ts]: Fixed this now

**Review by Mo-Lala84 — APPROVED** [review]
lgtm

**Pull request merged**